### PR TITLE
Assist chips

### PR DESCRIPTION
**Pull request by thiloSchlemmer**

### CODE REVIEW

File: src/modules/chips/components/assistChips/ElevationProfileChip.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const Update_Profile_Coordinates = 'update_profile_coordinates';
+
+
+/**
+ *
+ * @class
+ * @author thiloSchlemmer
+ */
+export class ElevationProfileChip extends AbstractAssistChip {
```
Comment (taulinger) [inline src/modules/chips/components/assistChips/ElevationProfileChip.js:15]: The component needs to overwrite `#onDisconnect` in order to ensure unsubscribing the observer (if we have one)

**Review by taulinger — COMMENTED** [review]
Did just a first quick look, think we're heading in the right direction.
Works already nice for me 👍

**Comment by taulinger** [comment]
The ChipsContainer test seems to be flaky:
```
Chrome Headless 109.0.5414.46 (Linux x86_64) absoluteChipsContainer.test.3095854908.js ChipsContainer when scroll buttons are clicked scrolls the container in the right and in the left direction FAILED
	Expected 32 to be less than 32.
	    at <Jasmine>
	    at UserContext.<anonymous> (/tmp/_karma_webpack_612838/commons.js:322384:33)
Chrome Headless 109.0.5414.46 (Linux x86_64): Executed 209 of 2623 (1 FAILED) (0 secs / 20.718 secs)
Chrome Headless 109.0.5414.46 (Linux x86_64) absoluteChipsContainer.test.3095854908.js ChipsContainer when scroll buttons are clicked scrolls the container in the right and in the left direction FAILED
	Expected 32 to be less than 32.
	    at <Jasmine>
	    at UserContext.<anonymous> (/tmp/_karma_webpack_612838/commons.js:322384:33)

```

**Comment by thiloSchlemmer** [comment]
I created #1447  to solve the flaky test

**Comment by taulinger** [comment]
One thing caught my eye: When a measurement or drawing geometry was updated the elevation profile won't be updated.

**Comment by thiloSchlemmer** [comment]
> One thing caught my eye: When a measurement or drawing geometry was changed the elevation profile won't be updated.

fixed with d5d6178564f76569b92ef84ec79ea288288bf059

**Review by taulinger — APPROVED** [review]

**Pull request merged**